src/components/MailingList/MailingList.jsx

```diff
@@ -55,11 +55,6 @@ class MailingList extends React.Component {
     render() {
         return (
             <EmailForm onSubmit={this.addEmailToList}>
-                <StaticP>
```

Comment:
I may replicate this change onto my branch, but again, it's the about page, so don't touch it on this branch since this one is meant for the contact page.